src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/Description.cs

```diff
@@ -0,0 +1,50 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using Microsoft.SqlTools.ServiceLayer.ExecutionPlan.ExecutionGraph;
+
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.Contracts
+{
+    /// <summary>
+    /// Holds all information that describes =the execution plan's graph.
+    /// </summary>
+    public class Description
+    {
+        /// <summary
+        /// The title for a graph
+        /// </summary>
+        public string Title { get; set; }
+        /// <summary>
+        /// Contains a string representation of the query the execution plan is for.
+        /// </summary>
+        public string QueryText { get; set; }
+        /// <summary>
+        /// The enabled clustered mode
+        /// </summary>
+        public string ClusteredMode { get; set; }
+        /// <summary>
+        /// Indicates if clustered mode is enabled on the server
+        /// </summary>
+        public bool IsClusteredMode { get; set; }
+        /// <summary>
+        /// List of missing indices for the graph
+        /// </summary>
+        public List<MissingIndex> MissingIndices { get; set; }
+
+        /// <summary>
+        /// Instantiates a DescriptionDTO
```

Comment:
You may not need this given Aasim's comment about, but fix these doc comments if you're keeping it